apps/reference/_supabase_js/generated/supabase-auth-admin-api.mdx

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
I think if we keep this page:
- Add an overview on what this API provides (but maybe this overlaps too much with the /docs/guides content)
- set `sidebar_label: Overview`